Javascript3/final-order/src/assets/Stylesheet.css

```diff
@@ -26,16 +26,13 @@ header {
     padding-top: 1.5vh;
```

Comment:
This file should ultimately be removed.

Reply:
Been and is in the process of doing so but be patient. Soon.